drivers/include/veml6070.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_veml6070 VEML6070
+ * @ingroup     drivers_sensors
+ * @brief       Device driver interface for the VEML6070 UV sensor
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the VEML6070 UV sensor.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef VEML6070_H
+#define VEML6070_H
+
+#include "saul.h"
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Integration times
+ */
+typedef enum veml6070_integrationtime {
+  VEML6070_HALF_T = 0,              /**< 1/2 T integration time */
+  VEML6070_1_T,                     /**< 1 T integration time */
+  VEML6070_2_T,                     /**< 2 T integration time */
+  VEML6070_4_T,                     /**< 4 T integration time */
+} veml6070_itime_t;
+
+/**
+ * @brief   Status and error return codes
+ */
+enum {
+    VEML6070_OK           =  0,     /**< everything was fine */
+    VEML6070_ERR_I2C      = -1,     /**< error initializing the I2C bus */
```

Comment:
I don't if it is convention in RIOT, but I'd rather have this positiv `1` and use `return -VEML6070_ERR_I2C` instead.